homeassistant/components/denonavr/media_player.py

```diff
@@ -74,58 +95,98 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
             unique_id = f"{config_entry.unique_id}-{receiver_zone.zone}"
         else:
             unique_id = None
```

Comment:
We can use the unique id of last resort here

https://developers.home-assistant.io/docs/entity_registry_index/#unique-id-of-last-resort

Reply:
ok, sounds like a plan 👍